src/Pages/ProductList/LeftNav/LeftNav.js

```diff
@@ -1,54 +1,49 @@
 import React from "react";
-import { withRouter, Link } from "react-router-dom";
+import "./LeftNav.scss";
+import LeftNav from "./LeftNavItem/LeftNavItem";
+import categoryData from "./../data/categoryData";
 
-const categoryData = [
-  "NEW",
-  "BEST",
-  "ONlY 29CM",
-  "상의",
-  "셔츠",
-  "바지",
-  "니트",
-  "신발",
-  "아우터",
-  "가방, 지갑",
-  "액세서리",
-  "아이웨어",
-  "주얼리",
-  "수영복",
-];
-
-class LeftNav extends React.Component {
+class MainLeftNav extends React.Component {
   render() {
+    const { gender } = this.props;
+
```

Comment:
```suggestion
  render() {
    const { gender } = this.props;
    const [menData, womenData] = this.categoryData
    const genderArr = gender === "MEN" ? menData : womenData
```
배열 역시 구조분해 할당이 가능합니다